storages/backends/dropbox.py

```diff
@@ -132,7 +139,7 @@ def _open(self, name, mode='rb'):
     def _save(self, name, content):
         content.open()
         if content.size <= self.CHUNK_SIZE:
-            self.client.files_upload(content.read(), self._full_path(name))
+            self.client.files_upload(content.read(), self._full_path(name), mode=WriteMode('overwrite'))
```

Comment:
The mode isn't constant. It needs to depend on what the user wants.

Reply:
django checks for exists, if it's true - it renames, so the mode does not actually metter

Reply:
@jschneier so, can I then add WriteMode as an option?

Reply:
@jschneier updated, made as an option